osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/deploymentspec/OnboardDAITask.java

```diff
@@ -77,16 +78,17 @@ public void executeTransaction(EntityManager em) throws Exception {
                 egressPort.setParentId(domainId);
                 if (domainId != null){
                 	//Element Object is not used in DefaultInstepctionPort for now, hence null
-                    controller.registerInspectionPort(new DefaultInspectionPort(ingressPort, egressPort, null));
+                    String redirectionTargetId = ds.getPortGroupLabel();
+                    Element element = controller.registerInspectionPort(new DefaultInspectionPort(ingressPort, egressPort,
+                                                                        null, redirectionTargetId));
+                    ds.setPortGroupLabel(element.getParentId());
```

Comment:
this should be set regardless of domainid being null. I would add after line 91 ds.setPortGroupLabel(element.getParentId());